admin/src/main.js

```diff
@@ -0,0 +1,63 @@
+import Vue from 'vue'
+import App from './App.vue'
+
+// without this async calls are not working
+import 'regenerator-runtime'
+
+import store from './store/store'
+
+import router from './router/router'
+import addNavigationGuards from './router/guards'
+
+import i18n from './i18n'
+
+import { ApolloClient, ApolloLink, InMemoryCache, HttpLink } from 'apollo-boost'
+import VueApollo from 'vue-apollo'
+
+import CONFIG from './config'
+
+import { BootstrapVue } from 'bootstrap-vue'
+
+const httpLink = new HttpLink({ uri: CONFIG.GRAPHQL_URI })
+
+const authLink = new ApolloLink((operation, forward) => {
+  const token = '' // store.state.token
```

Comment:
```suggestion
  const token = store.state.token
```

Please only disable this in debug mode. E.g. have a `NO_AUTH_FOR_DEVELOPMENT` .env setting. You can default it to `true` in the env (not in the `config/index.js`) at this given moment.